contracts/interfaces/handlers/IBosonPauseHandler.sol

```diff
@@ -38,4 +38,11 @@ interface IBosonPauseHandler is IBosonPauseEvents {
      * @param _regions - an array of regions to pause. See: {BosonTypes.PausableRegion}
      */
     function unpause(BosonTypes.PausableRegion[] calldata _regions) external;
+
+    /**
+     * @notice Returns the regions paused
+     *
+     * @return an array of regions that are currently paused. See: {BosonTypes.PausableRegion}
+     */
+    function getPausedRegions() external view returns (BosonTypes.PausableRegion[] memory);
```

Comment:
I think it would be nice to have a named return value. We do it in most parts of the protocol.